libs/MVS/Scene.cpp

```diff
@@ -1373,4 +1373,97 @@ bool Scene::ScaleImages(unsigned nMaxResolution, REAL scale, const String& folde
 	}
 	return true;
 } // ScaleImages
+
+/*----------------------------------------------------------------*/
+// detect region-of-interest based on camera positions, directions and sparse points
+// scale specifies the ratio of the ROI's diameter
+bool Scene::DetectROI(float scale)
+{
+    typedef CLISTDEF0IDX(Camera, uint32_t) CamArr;
+    CamArr camArr;
```

Comment:
even if I define sometimes a vector with the `Arr` termination, I usually call the variable with an `s` at the end, like in English